cardano-cli/src/Cardano/CLI/Shelley/Run/Governance.hs

```diff
@@ -177,3 +221,152 @@ runGovernanceUpdateProposal (OutputFile upFile) eNo genVerKeyFiles upPprams mCos
 
   firstExceptT ShelleyGovernanceCmdTextEnvWriteError . newExceptT $ writeFileTextEnvelope upFile Nothing upProp
 
+runGovernanceCreatePoll
+  :: Text
+  -> [Text]
+  -> Maybe Word
+  -> OutputFile
+  -> ExceptT ShelleyGovernanceCmdError IO ()
+runGovernanceCreatePoll govPollQuestion govPollAnswers govPollNonce out = do
+  let poll = GovernancePoll{ govPollQuestion, govPollAnswers, govPollNonce }
+
+  let description = fromString $ "An on-chain poll for SPOs: " <> Text.unpack govPollQuestion
+  firstExceptT ShelleyGovernanceCmdTextEnvWriteError . newExceptT $
+    writeFileTextEnvelope (unOutputFile out) (Just description) poll
+
+  let metadata = asTxMetadata poll
+        & metadataToJson TxMetadataJsonDetailedSchema
+
+  let outPath = unOutputFile out
+        & Text.encodeUtf8 . Text.pack
+
+  liftIO $ do
+    BSC.hPutStrLn stderr
+      "Poll created successfully.\n\
+      \Please submit a transaction using the resulting metadata.\n"
```

Comment:
Can this be converted to not use multiline string literals and instead, use `mconcat` like this?

```
    BSC.hPutStrLn stderr $ mconcat
      [ "Poll created successfully.\n"
      , "Please submit a transaction using the resulting metadata.\n"
      ]
```

Reply:
Yes, but why 😶 ? Is this the preferred coding convention in the cardano-node? I couldn't find any coding style guidelines.

Reply:
https://github.com/input-output-hk/cardano-node/wiki/Style-guide#multi-line-string-literals

Reply:
https://github.com/input-output-hk/cardano-node/compare/c018de7877eb6c513ddc029224ba2a189c4f6b2c..7adb11180e227089a5aa44845a06737efacf3401